dataduct/config/config_actions.py

```diff
@@ -32,7 +32,7 @@ def sync_from_s3(filename):
     text = s3_file.text
 
     if filename is None:
-        print text
+        raise ValueError('Filename for config sync must be provided')
```

Comment:
Should we really enforce this? Piping is a very valid use case, I did it when I tried this out. :P